diff --recursive -u xscreensaver-5.05/driver/xscreensaver.c xscreensaver-5.05.lio/driver/xscreensaver.c
--- xscreensaver-5.05/driver/xscreensaver.c     2008-04-07 05:05:12.000000000 +0100
+++ xscreensaver-5.05.lio/driver/xscreensaver.c 2008-04-07 17:24:44.000000000 +0200
@@ -832,7 +832,8 @@
   for (i = 0; i < count; i++)
     for (j = 0; j < i; j++)
       {
-        if (xsi[i].screen_number < 0) continue; /* already marked */
+        if (xsi[i].screen_number < 0 || xsi[j].screen_number < 0)
+            continue; /* already marked */

         *err = 0;
         if (X1 == X2 && Y1 == Y2 && W1 == W2 && H1 == H2)
